src/test/routes/work-address.test.ts

```diff
@@ -0,0 +1,31 @@
+import request from 'supertest';
+
+import { app } from '../../main/app';
+
+const PAGE_URL = '/work-address';
```

Comment:
Could import PageUrls for this?

Reply:
Thanks I've deleted this route, controller and tests, reusing place-of-work for this story instead.